system/interceptors/SystemAlertsInterceptor.cfc

```diff
@@ -0,0 +1,18 @@
+component extends="coldbox.system.Interceptor" {
+	property name="systemAlertsService" inject="delayedInjector:SystemAlertsService";
+
+	public void function configure() {}
+
+	public void function onApplicationStart() {
+		systemAlertsService.runStartupChecks();
+	}
+
+	public void function onClearSettingsCache( event, interceptData ) {
+		var category = interceptData.category ?: "";
+		if ( category == "dynamicform" ) {
```

Comment:
Could this go in the system config service where this interceptor is fired (after it) - rather than in interceptor?